README.rst

```diff
@@ -30,27 +15,26 @@ INSTALLING
 
 To install the core module, run:
 
-	make install-core
+	pip install python-graph
 
 To install the dot language support, run:
 
-	make install-dot
+	pip install python-graph[dot]
 
-Alternatively, if you don't have make, you can install the modules by running:
+If you want the development version, use poetry
 
-	./setup.py install
+	pip install poetry
```

Comment:
Look at the output, the 2 lines get squished to 1,
eventually make the dev install a separate heading